diff --git a/test/memory/lifetime/allocation.chpl b/test/memory/lifetime/allocation.chpl
index 55b3a4cfde..9691560381 100644
--- a/test/memory/lifetime/allocation.chpl
+++ b/test/memory/lifetime/allocation.chpl
@@ -4,9 +4,9 @@ use Time;
 // trial to not only 'warm up the cache' but also give jemalloc the chance to
 // warmup as well, allowing it to request more memory for the heap upfront rather
 // than on-demand.
-config const printTiming : bool;
-config const numTrials : int = 10;
-config const allocationsPerTrial : int = 1024 * 1024;
+config const printTiming : bool = true;
+config const numTrials : int = 3;
+config const allocationsPerTrial : int = 16 * 1024 * 1024;

 proc doUnmanagedAllocation() {
   var timer = new Timer();
diff --git a/test/memory/lifetime/allocation.execopts b/test/memory/lifetime/allocation.execopts
new file mode 100644
index 0000000000..3575947918
--- /dev/null
+++ b/test/memory/lifetime/allocation.execopts
@@ -0,0 +1 @@
+--printTiming=false --allocationsPerTrial=10
diff --git a/test/memory/lifetime/allocation.perfexecopts b/test/memory/lifetime/allocation.perfexecopts
deleted file mode 100644
index 68d65812fa..0000000000
--- a/test/memory/lifetime/allocation.perfexecopts
+++ /dev/null
@@ -1 +0,0 @@
---printTiming
